src/it/resources/org/checkstyle/suppressionxpathfilter/throwscount/SuppressionXpathRegressionThrowsCount1.java

```diff
@@ -0,0 +1,9 @@
+package org.checkstyle.suppressionxpathfilter.throwscount;
+
+interface SuppressionXpathRegressionThrowsCount1 {
```

Comment:
First input file should be class, second input file is an interface.

Third input class should be class with method, inside method anonymous class with method, and such method in annonimous class should be with violation

Reply:
means the second input is correct, right?
I have to change 1st and 3rd.

Reply:
Yes

Reply:
done